spampd.pl

```diff
@@ -949,6 +949,7 @@ ($$$)
 		'debug' => $debug,
 		'local_tests_only' => $options{'local-only'} || 0,
 		'home_dir_for_helpers' => $sa_home_dir, 
+		'user_dir' => (getpwnam($user))[7],
```

Comment:
Is there any chance of this `(getpwnam($user))[7]` introducing platform-specific (or otherwise undefined) behavior? Backwards compat. issues for people not using the FreeBSD port? I'm a bit weary here (and perhaps ignorant).  OTOH I know people have asked for this... perhaps yet another option?